src/app/app.component.html

```diff
@@ -1,6 +1,20 @@
 <head>
   <!--Import Google Icon Font-->
   <!--link href="http://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet"-->
+  <!--Global site tag (gtag.js) - Google Analytics -->
```

Comment:
I'm assuming this is for analytics access to? We've had to do this for a while, so awesome that you did it! Make sure Sam has access to the analytics page. Also, could you give me access as well?